actiontext/lib/templates/installer.rb

```diff
@@ -8,11 +8,17 @@
 JS_DEPENDENCIES = JS_PACKAGE["peerDependencies"].dup.merge \
   JS_PACKAGE["name"] => "^#{JS_PACKAGE["version"]}"
 
+if Rails::SourceAnnotationExtractor::Annotation.directories.include?("spec")
```

Comment:
This definitely can’t be right. I do think we’d have to look at generators and their hooks to get this right.

Reply:
@kaspth do you have an example I could learn from? Or can you further explain the kind of implementation you're expecting?